### DISCUSSION TITLE
Removing Issues tab on all georchestra "sub"-projects

### DISCUSSION THREAD

**Discussion by f-necas** [georchestra/georchestra#4267] [Ideas]
In order to avoid going into multiple projects, what if we remove all Issues tab in project settings :
![image](https://github.com/georchestra/georchestra/assets/39771412/034370fb-16db-4fc8-b96f-192330a4969f)

All issues would be created under georchestra/georchestra project.
They could be labeled, or something like that.

**Comment by f-necas** [comment]
An opnion on it @landryb @pmauduit @fvanderbiest @edevosc2c ?

**Reply by landryb** [reply]
i like creating separate issues for distinct subprojects (datadir, geonetwork, ...) because its easy to interlink them, and it conceptually makes sense to me to create an issue in a subproject when it only concerns this subproject... and there already have issues in most of them. But i'll adapt if you go this way :)

what is the goal ? being able to better use kanban projects ? cant they already work across projects ?

**Reply by edevosc2c** [reply]
I'm for keeping issues on each distinct GitHub repository. Having everything in one repo would be a big mess in my opinion.

If it's to have a global view of all the issues, you can achieve that through GitHub projects.

**Reply by f-necas** [reply]
The idea was to avoid duplicating milestones, labels, issues across projects to get "kanban projects" automated and simplified.

I tried to put a 24.1.0 milestone on  
- https://github.com/georchestra/geonetwork/pull/297
- https://github.com/georchestra/georchestra/issues/4268

And only georchestra's one ended up automatically in project board : https://github.com/orgs/georchestra/projects/18/views/2

**Reply by edevosc2c** [reply]
![image](https://github.com/georchestra/georchestra/assets/121870973/1484e705-ff6b-4c14-abb1-9dd9321ae4cb)

I can see two on my side.

**Reply by f-necas** [reply]
Yes but the second one was added manually

**Reply by edevosc2c** [reply]
What do you mean by manually? For an issue to appear on a GitHub project, it has to be added to the project.

The filter applies to what issues has been added to the project. Not on a global scale to all the GitHub repository across the GitHub organization.

**Comment by pmauduit** [comment]
> In order to avoid going into multiple projects, what if we remove all Issues tab in project settings

:+1:

**Comment by edevosc2c** [comment]
I see more downsides than benefits to have ALL the issues in one repo. One big downside is to have an ever increase of number of issues to the point that you feel you are overwhelmed by them.

Also, some users don't have anything to do with the other repositories. For instance, gabriel only works for georchestra gateway so he is subscribed to the notifications of georchestra-gateway only: https://github.com/georchestra/georchestra-gateway/watchers and not the georchestra main repo: https://github.com/georchestra/georchestra/watchers

If you move all the issues into the main georchestra repo, gabriel for example will have a hard time keeping up with the issues related to his project.

Anyway, if you want to understand how other projects deals with releases in a GitHub projects way, just look at how big open source projects do. Like newpipe: https://github.com/orgs/TeamNewPipe/projects?query=is%3Aclosed or yt-dlp: https://github.com/orgs/yt-dlp/projects?query=is%3Aopen or Kubernetes! : https://github.com/orgs/kubernetes/projects?query=is%3Aopen

**Comment by fvanderbiest** [comment]
As long as issues are specific to a geOrchestra module, they belong to this module IMO.
For issues which span several modules, it still makes sense to host them in the main repo.

**Comment by landryb** [comment]
and it makes it simpler to fix an issue in a given repo by a corresponding PR (eg linking issues & PRs in a repo instead of cross-repo)
